index.html

```diff
@@ -5705,6 +5717,62 @@ <h3>Complex Language Markup</h3>
 
     </section>
 
+    <section class="appendix normative">
+      <h2>Verification Algorithm</h2>
+
+      <p>
+This section contains an algorithm that <a>conforming verifier processors</a>
+can run when verifying a <a>verifiable credential</a> or a <a>verifiable
+presentation</a>. Any algorithm can be used that produces output that
```

Comment:
Yes, because of some dynamic loading thing that PR Preview doesn't support: https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1338.html#terminology ... We load terminology from a separate terms.html file, which PR Preview can't load (for some reason I haven't taken the time to debug).

Reply:
Yeah, PR Preview isn't good at handling multiple input files, and we've mostly lost its maintainer. Sorry for flagging something that's not actually a problem with this PR. :)